addons/automatic_updates.sh

```diff
@@ -18,29 +21,30 @@ debug_mode
 # Check if root
 root_check
 
-msg_box "This option will update your server every week on Saturdays at $AUT_UPDATES_TIME:00.
-The update will run the built in script '$SCRIPTS/update.sh' which will update both the server packages and Nextcloud itself.
-
-You can read more about it here: https://www.techandme.se/nextcloud-update-is-now-fully-automated/
-Please keep in mind that automatic updates might fail hence it's \
-important to have a proper backup in place if you plan to run this option.
-
-You can disable the automatic updates by entering the crontab file like this:
-'sudo crontab -e -u root'
-Then just put a hash (#) in front of the row that you want to disable.
-
-In the next step you will be able to choose to proceed or exit." "$SUBTITLE"
+# Check if automatic updates are already installed
+if ! crontab -u root -l | grep -q "$SCRIPTS/update.sh"
+then
+    # Ask for installing
+    install_popup "$SCRIPT_NAME"
+else
+    # Ask for removal or reinstallation
+    reinstall_remove_menu "$SCRIPT_NAME"
+    # Removal
+    crontab -u root -l | grep -v "$SCRIPTS/update.sh"  | crontab -u root -
+    sed -i "s|/sbin/shutdown -r +1||g" "$SCRIPTS"/update.sh
+    # Show successful uninstall if applicable
+    removal_popup "$SCRIPT_NAME"
+fi
 
-if yesno_box_yes "Do you want to enable automatic updates?"
+# Install automatic updates
+touch $VMLOGS/update.log
+crontab -u root -l | { cat; echo "0 $AUT_UPDATES_TIME * * 6 $SCRIPTS/update.sh minor >> $VMLOGS/update.log 2>&1"; } | crontab -u root -
+if yesno_box_yes "Do you want to reboot your server after every update? *recommended*"
 then
-    # TODO: delete the following line after a few releases. It was copied to the install-script.
-    nextcloud_occ config:app:set updatenotification notify_groups --value="[]"
-    touch $VMLOGS/update.log
-    crontab -u root -l | { cat; echo "0 $AUT_UPDATES_TIME * * 6 $SCRIPTS/update.sh minor >> $VMLOGS/update.log"; } | crontab -u root -
-    if yesno_box_yes "Do you want to reboot your server after every update? *recommended*"
-    then
-        sed -i "s|exit|/sbin/shutdown -r +1|g" "$SCRIPTS"/update.sh
-        echo "exit" >> "$SCRIPTS"/update.sh
-    fi
-    msg_box "Please remember to keep backups in case something should go wrong, you never know." "$SUBTITLE"
+    sed -i "s|exit|/sbin/shutdown -r +1|g" "$SCRIPTS"/update.sh
+    echo "exit" >> "$SCRIPTS"/update.sh
 fi
+
+msg_box "Please remember to keep backups in case something should go wrong, you never know."
```

Comment:
Please change from *should* to *would*.

Reply:
I've deleted the word since it sounds wrong